contracts/hackatom/tests/integration.rs

```diff
@@ -37,10 +37,12 @@ static WASM: &[u8] = include_bytes!("../target/wasm32-unknown-unknown/release/ha
 
 const DESERIALIZATION_LIMIT: usize = 20_000;
 
-fn make_init_msg() -> (InstantiateMsg, String) {
-    let verifier = String::from("verifies");
-    let beneficiary = String::from("benefits");
-    let creator = String::from("creator");
+fn make_init_msg<S: Storage + 'static, Q: Querier + 'static>(
+    deps: &Instance<MockApi, S, Q>,
+) -> (InstantiateMsg, String) {
```

Comment:
I'd just pass the `impl Api` as an argument, making the function signature much simpler.

Reply:
It'll have to be a `MockApi`, but I agree